### ISSUE TITLE
Wz list

### ISSUE THREAD

**Issue by anewsimpthe1st** [SmashMC-Development/Suggestions#907]
**Describe the suggestion**
I think there should be a separate list for when your in wz saying who's in during the time.

**Which servers does this apply to?**
Kyogre.

**Additional context**
N/A

**Comment by jantjewieger** [comment]
I don't think this should be a thing. That way when people see 1 specific person in warzone, who is very good, they will all just leave. Really takes away the competative side of the warzone

**Comment by WalkAway7** [comment]
I have to agree with wiepie here, I also don't think that a separate list would even be needed in the first place.

**Comment by BigChungv2** [comment]
i think it would be better if they removed the whos joining and leaving announcements, that way it brings the competitive element back, now whenever you load into wz a lot of people just leave if they think you're better than them or they don't wanna lose the legendary they just caught
